### PR TITLE
ENT-6200 Disable flaky test on Solaris 11, also

### PR DESCRIPTION
**Pull request by Lex-2008**
----

#

### CODE REVIEW

**Review by olehermanse — APPROVED** [review]

**Review by olehermanse — APPROVED** [review]

File: tests/acceptance/00_basics/ifelapsed_and_expireafter/timed/defaults.cf [hunk 1]
```diff
@@ -30,9 +30,9 @@ bundle agent init
   meta:
       # Backgrounding doesn't work properly on Windows or with fakeroot.
       # In addition, RHEL4 seems to have some problem killing the test
-      # afterwards, and HP-UX and Solaris 9 and 10 are too slow for timing
-      # to be accurate.
-      "test_skip_needs_work" string => "windows|using_fakeroot|redhat_4|hpux|sunos_5_9|sunos_5_10";
+      # afterwards, and HP-UX and Solaris are too slow for timing
+      # to be accurate. Also, this test randomly fails on SUSE 11 and 12.
```
Comment (vpodzime) [inline tests/acceptance/00_basics/ifelapsed_and_expireafter/timed/defaults.cf:34]: Please do not introduce (more) lies -- Solaris 11 is *not* too slow for timing to be accurate. There are genuine problems in our code or in LMDB.

**Review by vpodzime — CHANGES_REQUESTED** [review]

File: tests/acceptance/00_basics/ifelapsed_and_expireafter/timed/defaults.cf [hunk 2]
```diff
@@ -30,9 +30,9 @@ bundle agent init
   meta:
       # Backgrounding doesn't work properly on Windows or with fakeroot.
       # In addition, RHEL4 seems to have some problem killing the test
-      # afterwards, and HP-UX and Solaris 9 and 10 are too slow for timing
-      # to be accurate.
-      "test_skip_needs_work" string => "windows|using_fakeroot|redhat_4|hpux|sunos_5_9|sunos_5_10";
+      # afterwards, and HP-UX is too slow for timing to be accurate. Also,
+      # this test randomly fails on Soalris and SUSE 11 and 12.
```
Comment (vpodzime) [inline tests/acceptance/00_basics/ifelapsed_and_expireafter/timed/defaults.cf:34]: *Soalris*

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

**Review by vpodzime — APPROVED** [review]
ACK

**Comment by olehermanse** [comment]
Cherry-picks:
https://github.com/cfengine/core/pull/4360
https://github.com/cfengine/core/pull/4359

**Pull request merged**